### PR TITLE
Fix ci wheel building issue (CIBW env variables)

### PR DESCRIPTION
**Pull request by multiphaseCFD**
### Before submitting

Please complete the following checklist when submitting a PR:

- [ ] All new features must include a unit test.
      If you've fixed a bug or added code that should be tested, add a test to the
      [`tests`](../tests) directory!

- [ ] All new functions and code must be clearly commented and documented.
      If you do make documentation changes, make sure that the docs build and
      render correctly by running `make docs`.

- [ ] Ensure that the test suite passes, by running `make test`.

- [x] Add a new entry to the `.github/CHANGELOG.md` file, summarizing the
      change, and including a link back to the PR.

- [x] Ensure that code is properly formatted by running `make format`. 

When all the above are checked, delete everything above the dashed
line and fill in the pull request template.

------------------------------------------------------------------------------------------------------------

**Context:**

**Description of the Change:**

**Benefits:**

**Possible Drawbacks:**

**Related GitHub Issues:**

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,9 @@
 
 ### Bug fixes
 
+* Fix the CI X86-64 Linux wheels building.
```
Comment (mlxd) [inline .github/CHANGELOG.md:32]:
```suggestion
* Fix the CI environment variables for building wheels with the OpenMP backend.
```

**Review by mlxd — COMMENTED** [review]
Thanks @multiphaseCFD 
Just a quick wording suggestion on the Changelog

File: .github/workflows/wheel_linux_x86_64.yml [hunk 2]
```diff
@@ -76,8 +76,7 @@ jobs:
             source /opt/rh/devtoolset-11/enable -y
 
           CIBW_ENVIRONMENT: |
-            PATH=/opt/rh/devtoolset-11/root/usr/bin:$PATH
-            BACKEND=${{ matrix.exec_model }}
+            PATH=/opt/rh/devtoolset-11/root/usr/bin:$PATH BACKEND=${{ matrix.exec_model }}
```
Comment (AmintorDusko) [inline .github/workflows/wheel_linux_x86_64.yml:79]: Hi @multiphaseCFD, could you please explain why you needed to make this change?

Comment (multiphaseCFD) [reply .github/workflows/wheel_linux_x86_64.yml:79]: Hi @AmintorDusko , we need to separate the assignments by spaces to specify more than one environment variable.

File: .github/CHANGELOG.md (hunk 1 above)
Comment (multiphaseCFD) [reply .github/CHANGELOG.md]: Thanks @mlxd ! Done.

**Review by AmintorDusko — APPROVED** [review]
Thank you for that!

**Review by vincentmr — APPROVED** [review]
Looks good, thanks @multiphaseCFD

**Review by mlxd — APPROVED** [review]
Thanks @multiphaseCFD

**Pull request merged**